diff --git a/Cargo.toml b/Cargo.toml
index be61e3d..7087f17 100644
--- a/ripgrep/Cargo.toml
+++ b/ripgrep/Cargo.toml
@@ -61,8 +61,6 @@ serde_json = "1.0.23"
 termcolor = "1.1.0"
 textwrap = { version = "0.16.0", default-features = false }
 
-[target.'cfg(all(target_env = "musl", target_pointer_width = "64"))'.dependencies.tikv-jemallocator]
-version = "0.6.0"
 
 [dev-dependencies]
 serde = "1.0.77"
diff --git a/crates/core/main.rs b/crates/core/main.rs
index 46aac26..114b733 100644
--- a/ripgrep/crates/core/main.rs
+++ b/ripgrep/crates/core/main.rs
@@ -35,9 +35,8 @@ mod search;
 //
 // Moreover, we only do this on 64-bit systems since jemalloc doesn't support
 // i686.
-#[cfg(all(target_env = "musl", target_pointer_width = "64"))]
 #[global_allocator]
-static ALLOC: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
+static ALLOC: mimalloc::MiMalloc = mimalloc::MiMalloc;
 
 /// Then, as it was, then again it will be.
 fn main() -> ExitCode {
