diff --git a/src/context.rs b/src/context.rs
index dd531cb..22c1a37 100644
--- a/starship/src/context.rs
+++ b/starship/src/context.rs
@@ -518,9 +518,9 @@ impl DirContents {
 
         {
             let worker = move || {
-                let enumerated_dir = fs::read_dir(base).unwrap().enumerate();
+                let enumerated_dir = fs::read_dir(base).unwrap();
                 let _ = enumerated_dir
-                    .filter_map(|(_, entry)| entry.ok())
+                    .filter_map(|entry| entry.ok())
                     .try_for_each(|entry| tx.send(entry));
             };
